servicetalk-http-api/src/test/java/io/servicetalk/http/api/DefaultHttpCookiePairTest.java

```diff
@@ -41,24 +48,30 @@ public void testEqual() {
     public void testNotEqual() {
         // Name is case-sensitive:
         assertThat(new DefaultHttpCookiePair("foo", "bar"),
-                is(not(new DefaultHttpCookiePair("Foo", "bar"))));
+                   is(not(new DefaultHttpCookiePair("Foo", "bar"))));
         assertThat(new DefaultHttpCookiePair("foo", "bar").hashCode(),
-                is(not(new DefaultHttpCookiePair("Foo", "bar").hashCode())));
+                is(not(new DefaultHttpCookiePair("fooo", "bar").hashCode())));
+
+        assertThat(new DefaultHttpCookiePair("foo", "bar"),
+                   is(not(new DefaultHttpCookiePair(newAsciiString("Foo"), newAsciiString("bar")))));
+        assertThat(new DefaultHttpCookiePair("foo", "bar").hashCode(),
+                   is(not(new DefaultHttpCookiePair(newAsciiString("fooo"), newAsciiString("bar")).hashCode())));
+
```

Comment:
Checkstyle error in the build due to extra empty line.

Reply:
Fixed